### PR TITLE
Critical updates to articles in the DNS section - final articles

### PR DESCRIPTION
**Pull request by itsalyse**
Updates the following articles marked `critical`

Record Notes - https://docs.google.com/document/d/1FrD8mpF9A4jUjayEjfmqmCoeFJWLx8NKvc6cToPqNHE/edit

DNSSEC - https://docs.google.com/document/d/1TlU6usJ8Je7i29ZBQGp9UEWTdAmJfzOVkCmRmBYukN0/edit#heading=h.iun9sgf32gp

Updating NS Records for the Zone of a Hosted Domain - https://docs.google.com/document/d/1lGcNq7ENHlmq3GluXiuxkKRKvVrcWUPEsnSUM0xe9ww/edit#heading=h.4tfzkay4mi79

Auto-Importing DNS Records - https://docs.google.com/document/d/1lqWlJzVjZplaRS2orQTCOXGZVKwdK19m0d3cbKo2xIU/edit 

related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/zone-ns-records.md [hunk 1]
```diff
@@ -12,27 +12,29 @@ You can configure [NS records](/articles/ns-record/) at the apex of a zone of a
...
-1.  Switch to the <label>DNS</label> tab.
-
-    ![Domain DNS tab link](/files/domain-tab-dns-link.png)
+1.  Click the <label>Domain Names</label> tab, then locate and click the relevant domain to access the domain page.
+1. Click the <label>DNS</label> tab on the left side.
 1. Scroll down to locate the Zone NS records section.
-1. Click on <label>Edit zone NS records</label>
+1. Click <label>Edit zone NS records</label>
+
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/zone-ns-records.md:21]: https://drive.google.com/file/d/1bPRqAG1gjaNpSvcnPAcyi-6V3MOVLFx3/view?usp=sharing

File: content/articles/dnssec.md [hunk 2]
```diff
@@ -44,48 +44,50 @@ There are a variety of scenarios that DNSimple facilitates to ensure that your z
...
 <warning>
-If your domain is registered with another domain registrar, you should ideally remove the DS record from that registrar *before* removing the zone signing from DNSimple. Failure to remove the DS record within 48 hours of disabling DNSSEC will result in DNSSEC validation failures and will stop your domain from resolving with all DNSSEC-aware resolvers.
+If your domain is registered with another domain registrar, we strongly recommend removing the DS record from that registrar *before* removing the zone signing from DNSimple. Failure to remove the DS record within 48 hours of disabling DNSSEC will result in DNSSEC validation failures and will stop your domain from resolving with all DNSSEC-aware resolvers.
 </warning>
 
-To disable DNSSEC, go to the DNSSEC tab for the domain, and find the "Disable DNSSEC" card.
+To disable DNSSEC, click the **DNSSEC** tab for the domain, and locate the **Disable DNSSEC** card.
 
-![DNSSEC configured](/files/dnssec-configured.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dnssec.md:84]: https://drive.google.com/file/d/1umOWPdlt-Ax8vnfHrcyPLs3YBuUgaMmw/view?usp=sharing

File: content/articles/dnssec.md [hunk 3]
```diff
@@ -44,48 +44,50 @@ There are a variety of scenarios that DNSimple facilitates to ensure that your z
 
 DNSSEC is applied on a per-domain basis. You can manage DNSSEC by using the DNSSEC tab on a domain's management page.
 
-![Access DNSSEC through the DNSSEC tab on your domain management page](/files/dnssec-tab.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dnssec.md:47]: https://drive.google.com/file/d/1QfFsVE-fPZaDll3a_0t46bxG4QMRGxQe/view?usp=sharing

File: content/articles/dnssec.md [hunk 4]
```diff
@@ -19,86 +19,82 @@ categories:
...
+1. Click the **DNSSEC** tab on the left side.
+1. On the **Configure DNSSEC** card, click **Configure**.
 
 ![Configure DNSSEC](/files/dnssec-configure.png)
 
-Click on the "Enable DNSSEC" button.
+1. Click **Enable DNSSEC**.
 
-![Enable DNSSEC](/files/dnssec-enable.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dnssec.md:59]: https://drive.google.com/file/d/1V1sFCr50ZWuQHk8ovj6X_WVGBO3MkF01/view?usp=sharing

File: content/articles/dnssec.md [hunk 5]
```diff
@@ -110,15 +106,17 @@ During the key rotation, old and new keys are attached to your zone for 7 days (
...
-If your registrar requires the DNSKEY or other additional details, you can view your full DNSSEC configuration. First, find the "DNSSEC Configuration" card on the DNSSEC tab of a domain's management page.
+If your registrar requires the DNSKEY or other additional details, you can view your full DNSSEC configuration. 
+
+Locate the **DNSSEC Configuration** card under the **DNSSEC** tab of a domain's management page.
 
-Click on the "View Configuration" button.
+Click **View Configuration**.
 
-![Access DNSSEC through the DNSSEC tab on your domain management page](/files/dnssec-configured.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/dnssec.md:119]: https://drive.google.com/file/d/1tbvYD6H6pj7X4mZCXDWTP6WddMexHwU_/view?usp=sharing

**Review by stephanoodle — APPROVED** [review]

**Review by Pemacaa — APPROVED** [review]

**Comment by itsalyse** [comment]
@dperrymorrow the copy is finalized, and this is ready for screenshots

**Comment by dperrymorrow** [comment]
ok @itsalyse sorry for the delay, Ill try to get this done today

**Comment by dperrymorrow** [comment]
@itsalyse screenshots have been added please check out the preview url and let me know if it is 👌🏼

File: content/articles/record-notes.md [hunk 6]
```diff
@@ -24,13 +24,22 @@ When creating, updating, or deleting a DNS record from the [DNS record editor](/
 
 ## Creating a record note
 
-When you create or update a DNS record, you can fill in the _Notes_ field with the purpose of the record.
+1. Click the relevant domain in your Domain List. 
+1. Click the DNS badge in the top right.
+
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/record-notes.md:30]: https://drive.google.com/file/d/17o6ZQjxmiinRWoRFjj29CA7kquKBgKuP/view?usp=sharing

File: content/articles/record-notes.md [hunk 7]
```diff
@@ -24,13 +24,22 @@ When creating, updating, or deleting a DNS record from the [DNS record editor](/
...
+   
+1. To add a note to an existing record, click the **Add Note** icon on the right side, and enter your note.
+ 
+   <!--- needs screenshot -->
+
```
Comment (itsalyse) [inline content/articles/record-notes.md:35]: https://drive.google.com/file/d/1TTsWymQKB1hpqoY1oDKcssXTNeBRZEoF/view?usp=sharing

File: content/articles/record-notes.md (hunk 6 above)
Comment (itsalyse) [reply content/articles/record-notes.md:30]: @dperrymorrow

File: content/articles/record-notes.md (hunk 7 above)
Comment (itsalyse) [reply content/articles/record-notes.md:35]: @dperrymorrow

**Comment by itsalyse** [comment]
@dperrymorrow the dnssec article looks good, thanks! I added the record note screenshots in the comments so once those are up we should be good to merge

**Comment by dperrymorrow** [comment]
Ok i have updated the screenshots provided for record notes. Let me know if this is ready to push out live

**Comment by itsalyse** [comment]
@dperrymorrow the screenshots look good but the numbering is wonky again — is there a way I can update it that'll prevent that from happening when we add the screenshots?

File: content/articles/record-notes.md [hunk 8]
```diff
@@ -24,13 +24,22 @@ When creating, updating, or deleting a DNS record from the [DNS record editor](/
 
 ## Creating a record note
 
-When you create or update a DNS record, you can fill in the _Notes_ field with the purpose of the record.
+1. Click the relevant domain in your Domain List.
+1. Click the DNS badge in the top right.
+
+![screenshot: dns badge navigation item](/files/dns-badge.png)
+
+1. To add a note to an existing record, click the **Add Note** icon on the right side, and enter your note.
```
Comment (itsalyse) [inline content/articles/record-notes.md:32]: @dperrymorrow can you fix the numbering here and on 36? it's not showing up correctly in the preview

**Pull request merged**